src/main/java/ua/danit/final_project/controllers/LocationController.java

```diff
@@ -4,24 +4,33 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+import ua.danit.final_project.dto.DefaultMapper;
+import ua.danit.final_project.dto.LocationDto;
 import ua.danit.final_project.entities.Location;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'ua.danit.final_project.entities.Location'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=131&issues=AWY0HPGlM63is4RD-5Td&open=AWY0HPGlM63is4RD-5Td)

[ISSUE_KEY]: AWY0HPGlM63is4RD-5Td